packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -5,7 +5,7 @@
 import 'dart:async';
 import 'dart:collection';
 import 'dart:developer';
-import 'dart:ui' show AppLifecycleState;
+import 'dart:ui' show AppLifecycleState, window, Window;
```

Comment:
Is it possible for you to utilize a binding to get a window instance?  Unless absolutely necessary, developers should no longer directly import a window.

Reply:
In fact, this is a binding class itself.  Notice the window accessor method.  You should invoke the window accessor, and generally speaking, developers should never import `window` any more.